docs/guides/installation-helm.md

```diff
@@ -0,0 +1,78 @@
+This chart installs the [TriggerMesh Cloud Native Integration Platform](https://github.com/triggermesh/triggermesh) on a Kubernetes cluster.
+
+!!! note
```

Comment:
and maybe a note like this could use the following formatting

> 💡 **Note**: This is an alternative method of installation. A straight forward `kubectl apply` is actually all [it takes](installation.md) to get TriggerMesh up and running.

Reply:
aha!